### PR TITLE
Fix multiGPU Pytorch MovieLens

### PR DESCRIPTION
**Pull request by bschifferer**
Fix for https://github.com/NVIDIA-Merlin/NVTabular/issues/1290

### CODE REVIEW

**Review by karlhigley — APPROVED** [review]

**Comment by karlhigley** [comment]
rerun tests

File: examples/multi-gpu-movielens/01-03-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-PyTorch.ipynb [hunk 1]
```diff
@@ -633,23 +868,80 @@
...
+      "asyncio.exceptions.CancelledError\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1,1]<stdout>:Total batches: 488\n",
+      "[1,1]<stderr>:[2021-12-14 10:18:27.983077: E /tmp/pip-install-mwp6l21a/horovod_1f03263b83654efeb4c82a546f8aadfa/horovod/common/operations.cc:649] [1]: Horovod background loop uncaught exception: CUDA error: invalid configuration argument\n",
+      "[1,1]<stderr>:CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.\n",
```
Comment (jperez999) [inline examples/multi-gpu-movielens/01-03-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-PyTorch.ipynb:899]: These are errors right? Do we want this showing this?

**Review by jperez999 — CHANGES_REQUESTED** [review]

**Review by jperez999 — APPROVED** [review]

**Review by benfred — APPROVED** [review]

**Pull request merged**